source/iface/platform.rst

```diff
@@ -177,3 +162,19 @@ the type of information.
    * - extensions
      - vector_class<string_class>
      - Extension names supported by the platform
+
+
+.. _platform-example:
+
+.. rubric:: Example
```

Comment:
```suggestion
=======
Example
=======
```

It makes sense to move the example to the bottom and make the doc easy to scroll. The rubric creates a separation without creating a TOC entry. If we move to the bottom it should be its own section so you can navigate to the example by clicking in the column on the right.